src/tools/opt-dist/src/bolt.rs

```diff
@@ -76,8 +78,6 @@ pub fn bolt_optimize(path: &Utf8Path, profile: &BoltProfile) -> anyhow::Result<(
         // Inline functions smaller than 32 bytes
         .arg("-inline-small-functions")
         .arg("-inline-small-functions-bytes=32")
-        // Perform peephole optimizations
-        .arg("-peepholes=all")
```

Comment:
Is it useless?

Reply:
Based on Kobzol comments:

> we tried the peephole flag several times, and it never showed any performance improvement

If it's already tried, I won't test it again on Rust's infra. (And sadly I'm having problems of benchmarking it in my own servers)

Reply:
The Android maintainers (that use BOLT in their own Rust toolchain build) had some success with it, but we weren't able to replicate it.